client/state/selectors/get-billing-transaction-app-filter-values.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { groupBy, map } from 'lodash';
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import getPastBillingTransactions from './get-past-billing-transactions';
+import getUpcomingBillingTransactions from './get-upcoming-billing-transactions';
+
+/**
+ * Based on the transactions list, returns metadata for rendering the app filters with counts
+ *
+ * @param  {Object}  state           Global state tree
+ * @param  {String}  transactionType Transaction type
+ * @return {Array}                   App filter metadata
+ */
+export default createSelector(
+	( state, transactionType ) => {
+		const transactions =
+			'upcoming' === transactionType
```

Comment:
Seems like the following bit is used widely in the newly introduced selectors:

```
'upcoming' === transactionType
	? getUpcomingBillingTransactions( state )
	: getPastBillingTransactions( state );
```

Perhaps we could make a helper for it, or even a separate selector? Could help shorten the code just a little bit.